### PR TITLE
Give the XML Editor tab a title, reorganize the code, remove unnecess…

### LINKED ISSUE #807: Missing tab name (title)
QA Measures->XML Editor. Add tab title(XML Editor)

### PR DESCRIPTION
**Pull request by Jdew192837**
- Removed <title> tag since it doesn't display (it's iframe); If important I can add it back in
- Removed unused <style> tag
- Reorganized code for readability (just some indentation)

Fixes #807

### CODE REVIEW

**Comment by Jdew192837** [comment]
Before:
=====
xmleditorbefore

After:
====
xmleditorafter

**Comment by Jdew192837** [comment]
@teryhill PTAL :)

File: modules/MIPS/xmleditor/index.html [hunk 1]
```diff
@@ -1,35 +1,30 @@
...
-</head>
-<body>
-    <header>
-        <h1>XML Editing</h1>
-        
-        <i></i><b></b><u></u><s></s>
-    </header>
+    <head>
+        <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+        <span class="title" style="display: none;">XML Editor</span>
```
Comment (teryhill) [inline modules/MIPS/xmleditor/index.html]: Translation Here

File: modules/MIPS/xmleditor/index.html [hunk 2]
```diff
@@ -1,35 +1,30 @@
...
+        <link rel="stylesheet" href="screen.css">
+    </head>
+    <body>
+        <header>
+            <h1>XML Editor</h1>
```
Comment (teryhill) [inline modules/MIPS/xmleditor/index.html]: and here

**Comment by teryhill** [comment]
@Jdew192837 let me know when you get the translation fixed

File: modules/MIPS/xmleditor/index.html [hunk 3]
```diff
@@ -1,35 +1,30 @@
...
-    </script>
-</body>
-</html>
+            doctored.init("#editor1", {
+                onload: callback,
+                id:     "editor1" // unique id per domain used for saving data (localStorage key). Doesn't need to be the same as the element Id.
+            });
+        </script>
+    </body>
+</html>
```
Comment (aethelwulffe) [inline modules/MIPS/xmleditor/index.html]: If you change a file, add the new line ending for normalization.

**Comment by teryhill** [comment]
Art are you saying to hit enter on the last line of the program?
    On Thursday, December 7, 2017, 2:12:02 PM EST, Art Eaton <notifications@github.com> wrote:  
 
 
@aethelwulffe approved this pull request.

In modules/MIPS/xmleditor/index.html:
>  
-        
-    </script>
-</body>
-</html>
+            doctored.init("#editor1", {
+                onload: callback,
+                id:     "editor1" // unique id per domain used for saving data (localStorage key). Doesn't need to be the same as the element Id.
+            });
+        </script>
+    </body>
+</html>

If you change a file, add the new line ending for normalization.

**Comment by aethelwulffe** [comment]
Yes.

Don't ask me why.  GIT just likes it....

File: modules/MIPS/xmleditor/index.html [hunk 4]
```diff
@@ -1,35 +0,0 @@
-<!DOCTYPE html>
```
Comment (teryhill) [inline modules/MIPS/xmleditor/index.html]: Did you do a search to make sure that index.html is not called by another program?

**Comment by Jdew192837** [comment]
Yep @teryhill all I saw were AngularJS examples. :)

**Comment by teryhill** [comment]
@aethelwulffe  @tmccormi this looks good to me

**Comment by aethelwulffe** [comment]
@Trodrige or @tmccormi Merge when sane to do so.

**Pull request merged**